xml/libvirt_configuration_virsh.xml

```diff
@@ -0,0 +1,1231 @@
+    <para>
+     The following example of network definition creates a pool of all VFs
+     for the <xref linkend="vt.io.sriov"/> device with its physical function
+     (PF) at the network interface eth0 on the host:
+    </para>
+<screen>&lt;network&gt;
+  &lt;name&gt;passthrough&lt;/name&gt;
+    &lt;forward mode='hostdev' managed='yes'&gt;
+      &lt;pf dev='eth0'/&gt;
+    &lt;/forward&gt;
+  &lt;/network&gt;</screen>
+    <para>
+     To use this network on the host, save the above code to a file, for
+     example <filename>/tmp/passthrough.xml</filename>, and execute the
+     following commands. Remember to replace eth0 with the real network
+     interface name of your <xref linkend="vt.io.sriov"/> device's PF:
+    </para>
+<screen><prompt>tux &gt; </prompt><command>sudo virsh net-define /tmp/passthrough.xml</command>
+<prompt>tux &gt; </prompt><command>sudo</command> virsh net-autostart passthrough
+<prompt>tux &gt; </prompt><command>sudo</command> virsh net-start passthrough</screen>
+   </sect3>
+   <sect3 xml:id="libvirt.config.io.pool.guest">
+    <title>Configuring VM Guest to Use VF from the Pool</title>
+    <para>
+     The following example of guest device interface definition uses a VF of
+     the <xref linkend="vt.io.sriov"/> device from the pool created in
+     <xref linkend="libvirt.config.io.pool.host"/>. <systemitem class="library">libvirt</systemitem> automatically
+     derives the list of all VFs associated with that PF the first time the
+     guest is started.
+    </para>
+<screen>&lt;interface type='network'&gt;
+  &lt;source network='passthrough'&gt;
+&lt;/interface&gt;</screen>
+    <para>
+     To verify the list of associated VFs, run <command>virsh net-dumpxml
+     passthrough</command> on the host after the first guest that uses the
+     network with the pool of VFs starts.
+    </para>
+<screen>&lt;network connections='1'&gt;
+  &lt;name&gt;passthrough&lt;/name&gt;
+  &lt;uuid&gt;a6a26429-d483-d4ed-3465-4436ac786437&lt;/uuid&gt;
+  &lt;forward mode='hostdev' managed='yes'&gt;
+    &lt;pf dev='eth0'/&gt;
+    &lt;address type='pci' domain='0x0000' bus='0x02' slot='0x10' function='0x1'/&gt;
+    &lt;address type='pci' domain='0x0000' bus='0x02' slot='0x10' function='0x3'/&gt;
+    &lt;address type='pci' domain='0x0000' bus='0x02' slot='0x10' function='0x5'/&gt;
+    &lt;address type='pci' domain='0x0000' bus='0x02' slot='0x10' function='0x7'/&gt;
+    &lt;address type='pci' domain='0x0000' bus='0x02' slot='0x11' function='0x1'/&gt;
+    &lt;address type='pci' domain='0x0000' bus='0x02' slot='0x11' function='0x3'/&gt;
+    &lt;address type='pci' domain='0x0000' bus='0x02' slot='0x11' function='0x5'/&gt;
+  &lt;/forward&gt;
+  &lt;/network&gt;</screen>
+   </sect3>
+  </sect2>
+ </sect1>
+
+ <sect1 xml:id="libvirt.config.storage.virsh">
+  <title>Configuring Storage Devices</title>
+  <para>
+  Information about storage devices is defined under the <tag>disk</tag> element.
```

Comment:
'Information is defined' ... AFAIK, information is a plural word, maybe rephrase to: 'Storage devices are defined within the ```disk``` element.'